src/DBFTPlugin/DBFTPlugin.cs

```diff
@@ -19,23 +19,16 @@ protected override void Configure()
             Settings.Load(GetConfiguration());
         }
 
-        protected override void OnPluginsLoaded()
+        protected override void OnSystemLoaded(NeoSystem system)
         {
-            walletProvider = GetService<IWalletProvider>();
-            if (Settings.Default.AutoStart)
-                walletProvider.WalletOpened += WalletProvider_WalletOpened;
```

Comment:
The `AutoStart` logic is deleted?

Reply:
please check https://github.com/neo-project/neo/pull/2344